server/rest/src/test/java/org/infinispan/rest/resources/CacheManagerResourceTest.java

```diff
@@ -86,7 +86,8 @@ public void testHealth() {
 
       Json jsonNode = Json.read(response.getBody());
       Json clusterHealth = jsonNode.at("cluster_health");
-      assertEquals(clusterHealth.at("health_status").asString(), "HEALTHY");
+      // One of the caches is in FAILED state
```

Comment:
would be better to use the enum on tests as well

Reply:
fixed